service/src/main/java/bio/terra/pipelines/app/controller/GlobalExceptionHandler.java

```diff
@@ -41,7 +41,6 @@ public ResponseEntity<ApiErrorReport> errorReportHandler(ErrorReportException ex
   // -- validation exceptions - we don't control the exception raised
   @ExceptionHandler({
     MethodArgumentTypeMismatchException.class,
-    HttpRequestMethodNotSupportedException.class,
```

Comment:
this makes me a little nervous. we don't have any instances of this error in Sentry in dev in the last 30 days but this would be like a user doing something wrong, so not sure if this is going to mask something that the user is doing. but I think that it'll be ok - if that's the case, the user will still get the 4xx response and we can investigate logs if it's confusing to them

Reply:
my understanding is if we keep it, those errors will always be caught by this rather than later on and we'll still get sentry alerts for it. unless we move the sentry filtering thing into this method, maybe that's the answer?

Reply:
ah i misunderstood, you're totally right - will revert this